build-support/bin/release.sh

```diff
@@ -559,6 +561,7 @@ function adjust_wheel_platform() {
   local src_plat="$1"
   local dst_plat="$2"
   local dir="$3"
+  # shellcheck disable=SC2044
```

Comment:
It's not clear to me why this triggers from reading https://github.com/koalaman/shellcheck/wiki/SC2204?

Reply:
(so: this disable is fine!)

Reply:
Think you're looking at the wrong link! Refer to https://github.com/koalaman/shellcheck/wiki/SC2044.

Reply:
Ah! Well then this could be changed into:
```bash
find "$dir" -type f -name "*${src_plat}.whl" | while read -r src_whl; do
  # ...
done
```
perhaps?

Reply:
(and I would recommend this, as `for` loops are generally extremely fragile unless it's from a controlled input like `seq`!)

Reply:
Will try out. Thanks!